bin/p2-preparer/main.go

```diff
@@ -44,6 +44,12 @@ func main() {
 		defer statusServer.Close()
 	}
 
+	if preparerConfig.RequireFile != "" {
```

Comment:
oh, I didn't know that p2-exec starts p2-preparer

then, in the design doc, what is the point of having the p2-preparer reading from config at all? If it's just going to pass through p2-exec?